src/env_dispatch.cpp

```diff
@@ -531,6 +531,15 @@ static void init_curses(const environment_t &vars) {
     curses_initialized = true;
 }
 
+static const std::string utf8_locales[] = {
```

Comment:
This could be `static const char *utf8_locales[]` to save some allocations. The set location would be `setlocale(LC_CTYPE, loc)`

Reply:
I tried something like that and gcc complained about ISO-C++ not allowing it. Maybe I did something wrong, will try again!

(also this list is super cheesy, but I didn't want to add the full 300 or so utf-8 locales that glibc on my system knows)